modules/EvseManager/EvseManager.cpp

```diff
@@ -634,6 +634,26 @@ void EvseManager::ready() {
             ev_info = types::evse_manager::EVInfo();
             p_evse->publish_ev_info(ev_info);
         }
+
+        if (s == types::evse_manager::SessionEventEnum::SessionStarted &&
+            charger->getSessionStartedReason() == types::evse_manager::StartSessionReason::Authorized) {
+            Array payment_options;
+            payment_options.insert(payment_options.end(), "ExternalPayment");
+            r_hlc[0]->call_set_PaymentOptions(payment_options);
```

Comment:
we should check if hlc is enabled / connection is available

Reply:
Added